packages/palette/package.json

```diff
@@ -120,6 +120,7 @@
     "d3-shape": "^1.3.5",
     "luxon": "^1.15",
     "rc-slider": "^8.6.2",
+    "react-focus-on": "^3.4.1",
```

Comment:
So, I'm always reluctant to introduce any UI-specific dependency. Here I think it makes sense because doing all of this and doing it correctly in a cross-platform way is super tricky.

Alternatively if we wanted to punt on the a11y stuff for now, we can fix/improve the scroll locking and still extract the ModalBase, which would solve the immediate issue at hand.

Reply:
Yeah, understand that. 

Just to document the decision, did you consider any alternatives? Was there a specific reason for choosing this dep?

Reply:
* One possible alternative is [react-modal](https://reactcommunity.org/react-modal/) — which I think makes too many assumptions about your UI, and is a single-purpose dependency (it's just a modal).
* Alternatives for something as "simple" as scroll-locking are hit and miss. For instance the popular [body-scroll-lock](https://github.com/willmcpo/body-scroll-lock) will break under certain SSR environments currently.
* react-focus-on is a composition of a couple of different packages, each of which is independently useful (see [first paragraph in README](https://github.com/theKashey/react-focus-on))
* All of those sub-packages are by the same author/of similar quality/include type definitions
* react-focus-on also supports scattered locks through it's `shards` API